docs/en/setup/service-agent/php-agent/README.md

```diff
@@ -52,6 +52,9 @@ way:
 
    ```shell
    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain none
+   # Optional
+   rustup install stable
+   rustup default stable
```

Comment:
Installing stable is useless, because the toolchain will be override by `rust-toolchain.toml`.

Reply:
However, do not execute it; instead, output 'rustc' not found.

Reply:
Maybe execute `rustup show` will install the overrided toolchain.

Reply:
I known why you have to install stable, because the `rust-toolchain.toml` is forgot to add in `config.m4`.

<https://github.com/apache/skywalking-php/pull/88/commits/cba111f7b7fe394fd373fdcd81629bf0873e7a68>